hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncTool.java

```diff
@@ -227,6 +227,9 @@ public static void main(String[] args) {
     }
     FileSystem fs = FSUtils.getFs(cfg.basePath, new Configuration());
     HiveConf hiveConf = new HiveConf();
+    if (cfg.hiveMetaStoreUri != null) {
+      hiveConf.set("hive.metastore.uris",cfg.hiveMetaStoreUri);
```

Comment:
Where would we use this config option: `hive.metastore.uris`? 
And add white space between `",cfg`.

Reply:
This configuration is needed when we are synchronizing to remote hive and i will add this  white space

Reply:
If you want to split the tasks, then add it in the subsequence PR. We should make each change reasonable. No assumptions should be made about the future.

Reply:
To be right, I have also developed and tested the subtasks. Should I submit a new pr, or merge into this pr ？

Reply:
If you want to remain this config option, it would be better to add more logic about its usage. That means it would be better to merge that PR into this one.